### PR TITLE
[Rating#589] Single star view

### PR DESCRIPTION
**Pull request by michael-zimmermann**
As a preparation for the rating view, a single configurable rating star is implemented.
This star wil be used for the RatingView and the RatingInputControl.
The star is configurable. The configuration possibles are added to the demo.

Simulator Screenshot - iPhone 15 - 2023-11-08 at 16 40 57

### CODE REVIEW

File: .gitmodules
Comment (jacklyn-situmorang) [inline .gitmodules:1]: Is this change meant to be included in this PR?

**Review by jacklyn-situmorang — COMMENTED** [review]
Looks good, I'm just wondering if the change on .gitmodules is intended

File: .gitmodules
Comment (michael-zimmermann) [reply .gitmodules:1]: Very good point, I don't thinks so

Comment (michael-zimmermann) [reply .gitmodules:1]: I've removed it from the PR.

**Review by jacklyn-situmorang — APPROVED** [review]

**Pull request merged**